README.rst

```diff
@@ -31,24 +31,26 @@
 What is Data.Rentgen?
 ---------------------
 
-Data.Rentgen is a DataLineage service compatible with `OpenLineage <https://openlineage.io/>`_ specification.
+Data.Rentgen is a Data Motion Lineage service, compatible with `OpenLineage <https://openlineage.io/>`_ specification.
 
-**Note**: service is under active development, and is not ready to use.
+**Note**: service is under active development, and is not ready to use yet.
 
 Goals
 -----
 
-* Collect lineage events produced by OpenLineage clients & integrations (Spark, Airflow, Flink, custom ones).
+* Collect lineage events produced by OpenLineage clients & integrations (Spark, Airflow).
+* Support consuming large amounts of lineage events, by using Kafka as event buffer and storing data in tables partitioned by event timestamp.
 * Store operation-grained events (instead of job grained `Marquez <https://marquezproject.ai/>`_), for better detalization.
-* Provide API for run ↔ dataset lineage, as well as parent run → children run lineage.
-* Support handling large amounts of lineage events, using Kafka as event buffer and storing data in tables partitioned by event timestamp.
+* Provide API for building run ↔ dataset lineage, as well as parent run → children run lineage.
+* Ability to build lineage graph with specific time boundaries (unlike Marquez there lineage is build only for last job run).
```

Comment:
Maybe move all compares to separate section?

Reply:
I mean mb better make a compare table in the README ?

Reply:
We'll probably add this comparison later